src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -733,6 +749,28 @@ public function collectFromCode($content, $fileName, Module $module)
                     continue;
                 }
 
+                // Allow _t(Entity.Key, 'translation', $var) and expand _t(Entity.Key, $var)
+                if ($id == T_VARIABLE && !empty($currentEntity)) {
+                    // We have a translation, eg: _t(Entity.Key, 'translation', $var)
+                    if (count($currentEntity) == 2) {
+                        continue;
+                    }
+
+                    // The variable is the second argument or present in the parameter
```

Comment:
What is "the parameter" in this context? Does that mean it's in the third (or subsequent) argument?

Reply:
it's when you have _t('key', 'translation', $var) or _t('key', 'translation', 'comments' . $var) => not than this should be ever done anyway, but that can happen

Reply:
i'm not sure how to phrase that correctly :-)

Reply:
Reading again I think it's probably fine as is